binding/nodejs/index.test.js

```diff
@@ -19,16 +19,38 @@ const { PvArgumentError, PvStateError } = require("./errors");
 const { getPlatform, getSystemLibraryPath } = require("./platforms");
 
 const MODEL_PATH = "./lib/common/rhino_params.pv";
+const MODEL_PATH_DE = "../../lib/common/rhino_params_de.pv";
```

Comment:
Is this intentional? I'm surprised to see the EN models at a different path to the non-EN models.

Reply:
the EN model was available locally and I didn't change its path. But other models are not, so I used a different path for them.

Reply:
I'm not overly familiar with the nodejs package but I guess that means we don't package the non-EN models? Not saying we should, but something to note for the future I suppose.